utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
+# uses subproccess instead, more secure.
+command =  ["git", "log", "--summary", "--stat", "--no-merges", "--date=short", rev_range]
+
+fin = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+output, error = process.communicate()
```

Comment:
The variable names are wrong (also detected by the CI).

Reply:
fixed in the latest commit!